backend/api/v1/v1_forms/tests/tests_form_submission.py

```diff
@@ -105,4 +105,4 @@ def test_create_new_submission(self):
         data = data.json()
         self.assertEqual(
             data,
-            {"message": "This field is required.|administration is required."})
+            {"message": "name is required.|administration is required."})
```

Comment:
Can we add a successful response for test_form_submission by updating the payload and reposting?

Reply:
Already there, line number 96